### ISSUE TITLE
Disable Pet Pickup in MobArena

### ISSUE THREAD

**Issue by Akiranya** [xXKeyleXx/MyPet#1213]
Please provide the following info.

Please use the `/petadmin ticket` command and attach the generated `ticket.zip` to this issue.
[ticket.zip](https://github.com/xXKeyleXx/MyPet/files/2829321/ticket.zip)

**Bukkit/Spigot/Bungee Build/Version:**
`This server is running Paper version git-Paper-513 (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)`

**MyPet Build/Version:**
`Dev #733`

**Do you use the premium version? (this wont change anything for you)**
yes

**What is the expected result?**
Can you add a config that turns off pet pickup in MobArena? :)
Players could steal powerful items from MobArena using pet pickup...because I created some interesting redstone mechines where players could get supply items like enchanted golden apples, etc.

**What happens instead of that?**
Players were stealing supply items designed for MobArena only, using pet pickup. They took these items back to their survival mode, which was imbalanced. What genius for seeking bugs they are :0

btw I understand that I could disable mypet completely in MobArena to solve this problem, but that could lose bit of fun... Players really enjoyed fighting against monsters in MobArena with their pets.

**Error:**
None

**Comment by xXKeyleXx** [comment]
This is what the extended permissions are for. If these are not enough or don't work feel free to ask again.

**Comment by Akiranya** [comment]
Thank you it works now! I didn't thought it could be done with permission control, because it's much easier than I thought before. Pet pickup is disabled automatically when players join mobarena with their pet pickup on.

But I found another issue. Players cannot feed or ride their pets when they have left arena. Seems like they jsut cannot use the funtions which I have disabled for mobarena. Other functions work normally. Luckily, I also noticed that they can fix the issue by reconnecting server. Maybe this is just a bug with MobArena, not MyPet... I just can't figure it out.
